can-bus/json/BMS/BMS_tx.json

```diff
@@ -1,10 +1,16 @@
 {
-    "BMS_Test1": {
+    "BMS_OkStatuses": {
```

Comment:
We're going to have a hell of a time with the inevitable merge conflicts on some of these json files

Reply:
Yeah agreed
This PR changed a lot of the JSON CAN too, maybe we'll get your PR in first and I can fix any conflicts after